smooks-all/pom.xml

```diff
@@ -257,6 +257,12 @@
             <artifactId>mockito-all</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I thought `junit` was auto added as a dependency from the parent POM, or did that change in the pom reshuffle from a while ago?